.github/labeler.yml

```diff
@@ -1,3 +1,4 @@
-possible stability impact:
-- libfdo-data/fdo_data.h
-- libfdo-data/libfdo-data-go.doc
+"possible stability impact":
```

Comment:
Again a separate commit, generally each fix should be independent, make it easier for bisect and review.